### PR TITLE
Fixed global rate limit not updating after a reload 

### PR DESCRIPTION
**Pull request by lonelycode**
related to #356 fixes bug in PR #1138

### CODE REVIEW

File: mw_api_rate_limit.go [hunk 1]
```diff
@@ -32,7 +35,7 @@ func (k *RateLimitForAPI) EnabledForSpec() bool {
 	k.apiSess = &user.SessionState{
 		Rate:        k.Spec.GlobalRateLimit.Rate,
 		Per:         k.Spec.GlobalRateLimit.Per,
-		LastUpdated: "na",
+		LastUpdated: strconv.Itoa(int(time.Now().UnixNano())), // Set last updated on each load to ensure we always use a new rate limit bucket
```
Comment (mvdan) [inline mw_api_rate_limit.go]: Put the comment on its own line above to avoid one really long line

File: mw_api_rate_limit_test.go [hunk 2]
```diff
@@ -46,13 +47,14 @@ func getGlobalRLAuthKeyChain(spec *APISpec) http.Handler {
...
 	baseMid := BaseMiddleware{spec, proxy}
+	rlMW := &RateLimitForAPI{BaseMiddleware: baseMid}
 	chain := alice.New(mwList(
 		&IPWhiteListMiddleware{baseMid},
 		&AuthKey{baseMid},
 		&VersionCheck{BaseMiddleware: baseMid},
 		&KeyExpired{baseMid},
 		&AccessRightsCheck{baseMid},
-		&RateLimitForAPI{BaseMiddleware: baseMid},
+		rlMW,
```
Comment (mvdan) [inline mw_api_rate_limit_test.go]: Why the vars?

Comment (lonelycode) [reply mw_api_rate_limit_test.go]: Ah, I thought the init wasn't getting called, so I was calling them manually.

**Comment by ilijabojanovic** [comment]
@lonelycode @mvdan approved for merge

**Pull request merged**